addlicense/main.go

```diff
@@ -242,6 +242,7 @@ func processFile(f *file, t *template.Template, license LicenseData, checkonly b
 			return err
 		}
 		if lic == nil { // Unknown fileExtension
+			logger.Printf("[DEBUG] skipping because of unknown extension: %s", f.path)
```

Comment:
I like this idea, but the current logging implementation needs some rework, so this will currently print out a large number of debug noise during regular noise.

Do you mind if we remove it for this PR so we can ship the changes and log it as a GitHub Issue instead?

Reply:
Sure. I created an issue: https://github.com/hashicorp/copywrite/issues/87